apps/judicial-system/backend/src/app/modules/notification/services/caseNotification/caseNotification.service.ts

```diff
@@ -2055,14 +2140,14 @@ export class CaseNotificationService extends BaseNotificationService {
       )
 
       promises.push(
-        this.sendEmail(
+        this.sendEmail({
           subject,
-          defenderHtml,
-          theCase.defenderName,
-          theCase.defenderEmail,
-          undefined,
-          !theCase.defenderNationalId,
-        ),
+          html: defenderHtml,
+          recipientName: theCase.defenderName,
+          recipientEmail: theCase.defenderEmail,
+          attachments: undefined,
```

Comment:
Consider removing undefined properties as they are no longer needed when passing in an object as a parameter.